auth-web/jest.config.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
   snapshotSerializers: [
     'jest-serializer-vue'
   ],
-  'setupFiles': ['jest-localstorage-mock'],
+  setupFiles: ['jest-localstorage-mock', 'dotenv/config'],
```

Comment:
We need this, so the runner in VSCODE doesn't fail loading environment variables